### PR TITLE
[KOGITO-3639] - Ignore Verify if the s2i build is finished as expected performing a native build with persistence enabled Scenario on kogito images

### PR DESCRIPTION
**Pull request by Kaitou786**
See: https://issues.redhat.com/browse/KOGITO-3639
Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by Kaitou786** [comment]
@spolti if it is okay with you we can merge this. So the nightly can succeed today :p

File: tests/features/kogito-quarkus-ubi8-s2i.feature [hunk 1]
```diff
@@ -119,6 +119,8 @@ Feature: kogito-quarkus-ubi8-s2i image tests
     And s2i build log should contain [persistence] Generated checksum for /home/kogito/data/protobufs/persons.proto with the name: /home/kogito/data/protobufs/persons-md5.txt
     And s2i build log should contain [persistence] Generated checksum for /home/kogito/data/protobufs/demo.orders.proto with the name: /home/kogito/data/protobufs/demo.orders-md5.txt
 
+  #ignore until https://issues.redhat.com/browse/KOGITO-3638 is resolved
```
Comment (spolti) [inline tests/features/kogito-quarkus-ubi8-s2i.feature:122]: KOGITO-3639?

**Review by spolti — CHANGES_REQUESTED** [review]

File: tests/features/kogito-quarkus-ubi8-s2i.feature (hunk 1 above)
Comment (Kaitou786) [reply tests/features/kogito-quarkus-ubi8-s2i.feature:122]: Nope, [3639](https://issues.redhat.com/browse/KOGITO-3639) is this one,(Ignore the test).
[3638](https://issues.redhat.com/browse/KOGITO-3638) is the one, created for further investigation on why this test is failing.

**Review by spolti — APPROVED** [review]

**Comment by radtriste** [comment]
@Kaitou786 chery-pick to 0.17.x ? :)

**Comment by Kaitou786** [comment]
@radtriste cherrypick #264

**Pull request merged**